examples/puppet_headnode/manifests/static/boot_notify.py

```diff
@@ -1,10 +1,10 @@
-#!/usr/bin/env python
+import re
+import os
+
+# !/usr/bin/env python
```

Comment:
Yeah, let's just get rid of it; for "real" scripts, there are more reliable ways of finding the right interpreter anyway (`scripts`/`entry_points` in setup.py), so for the examples let's just leave it out.